tests/queries/0_stateless/03254_parquet_bool_native_reader.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Tags: no-ubsan, no-fasttest
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+USER_FILES_PATH=$($CLICKHOUSE_CLIENT_BINARY --query "select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep Exception | awk '{gsub("/nonexist.txt","",$9); print $9}')
+
+WORKING_DIR="${USER_FILES_PATH}/${CLICKHOUSE_TEST_UNIQUE_NAME}"
+
+mkdir -p "${WORKING_DIR}"
+
+DATA_FILE="${CUR_DIR}/data_parquet/nullbool.parquet"
```

Comment:
I'd just do `INSERT INTO TABLE FUNCTION file('nullbool.parquet') SELECT base64Decode('....')` embedding data into test (since file is pretty small) and then work with `file('nullbool.parquet')`. It can be done even in `.sql` test